src/main/java/de/bentzin/ingwer/command/IngwerCommandManager.java

```diff
@@ -2,91 +2,108 @@
 
 import de.bentzin.ingwer.Ingwer;
 import de.bentzin.ingwer.command.ext.Permissioned;
-import de.bentzin.ingwer.features.Feature;
 import de.bentzin.ingwer.logging.Logger;
 import de.bentzin.ingwer.thow.IngwerThrower;
 import de.bentzin.ingwer.thow.ThrowType;
 import de.bentzin.tools.register.Registerator;
-import org.bukkit.command.Command;
 import org.jetbrains.annotations.Contract;
 import org.jetbrains.annotations.NotNull;
 
 public final class IngwerCommandManager extends Registerator<IngwerCommand> {
 
-    public static IngwerCommandManager getInstance() {
-        return Ingwer.getCommandManager();
-    }
-
-    private Logger logger;
+    private final Logger logger;
 
     public IngwerCommandManager() {
         this.logger = Ingwer.getLogger().adopt("CMD");
     }
 
+    public static IngwerCommandManager getInstance() {
+        return Ingwer.getCommandManager();
+    }
+
     public Logger getLogger() {
         return logger;
     }
 
     /**
-     *
      * @param newName
      * @return if name is already taken
      */
     public boolean checkName(String newName) {
         for (IngwerCommand command : this) {
-            if(command.getName().equalsIgnoreCase(newName)) {return true;}
+            if (command.getName().equalsIgnoreCase(newName)) {
+                return true;
+            }
         }
         return false;
     }
 
     public void preRunCommand(String input, IngwerCommandSender sender, @NotNull CommandTarget senderType) {
-        if(!senderType.isLast()) try {
+        if (!senderType.isLast()) try {
             throw new IllegalStateException("Unexpected value: " + senderType.name() + ". senderType cant be multi-reference!");
-        }catch (IllegalStateException e) { IngwerThrower.acceptS(e,ThrowType.COMMAND);}
+        } catch (IllegalStateException e) {
+            IngwerThrower.acceptS(e, ThrowType.COMMAND);
+        }
         //run
-        if(senderType.comesWithPrefix()) {
+        if (senderType.comesWithPrefix()) {
             if (input.startsWith(Ingwer.getPreferences().prefix() + "")) {
```

Comment:
### In case you don't know:

If you want to input a "CHAR" into a String requiring method you need to encapsulate it in a String with one of these methods:

1. char + "" -> used it here
2. String.valueOf(char)
3. new String(char)
4. new Character(char).toString()